lib/rspec/que.rb

```diff
@@ -6,6 +8,10 @@ def queue_up(job_class)
       Matchers::QueueUp.new(job_class)
     end
 
+    def queue_up_anything
```

Comment:
Thinking about ditching this now @cezarelnazli as the new method is better

Reply:
Yup, I think it's more in line with how RSpec works.